JSDemos/Demos/Popup/Overview/Angular/description.md

```diff
@@ -1,10 +1,34 @@
+This demo shows how to show and hide the Popup component, how to fill it with content, specify its position and other configurations. To get started with the DevExtreme Popup component, refer to the following tutorial for step-by-step instructions: [Getting Started with Popup](/Documentation/Guide/UI_Components/Popup/Getting_Started_with_Popup/).
 
-- **Content**        
-Put any valid markup inside the component. 
+## Show and Hide the Popup
 
-- **Title Bar**       
-Set [showTitle](Documentation/ApiReference/UI_Components/dxPopup/Configuration/#showTitle) to **true** and use the [title](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#title) property to specify the caption. 
+Use two-way binding to define the [visible](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#visible) property and bind it to a boolean variable, `popupVisible` in this demo. The `showInfo(employee)` method assigns `true` to the `popupVisible` variable, and the Popup becomes visible. You can close the Popup in multiple ways:
 
-- **Show and hide the Popup**      
-Use two-way binding to define the [visible](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#visible) property and manipulate its value directly. If the Popup should be hidden on outside clicks, enable the [hideOnOutsideClick](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#hideOnOutsideClick) property.
+- Built-in close button    
+Enable the [showCloseButton](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#showCloseButton) property to display the close button in the Popup's [title area](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#showTitle).
+
+- Custom close button    
+This demo shows how to add custom buttons to the Popup. One of them uses an [onClick](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#onClick) handler to set the `popupVisible` variable to `false`, and the Popup becomes invisible. To learn more, follow the **Configure the Popup** topic below.
+
+- On outside click    
+Users can also hide the Popup on outside clicks, if you enable the [hideOnOutsideClick](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#hideOnOutsideClick) property.
+
+## Configure the Popup
+
+The Popup inner area is divided into two parts: title area and content area.
+
+- Content area       
+To fill the Popup with content, put a markup inside the component. You can also specify a toolbar at the bottom of the content area. To do this, declare the [toolbarItems](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/) in the markup as shown in this demo. Set each item's [toolbar](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/#toolbar) property to `bottom`. To learn more about toolbar configuration, refer to the following tutorial: [Getting Started with Toolbar](/Documentation/Guide/UI_Components/Toolbar/Getting_Started_with_Toolbar/)
```

Comment:
"a markup" - the word "markup" is an uncountable noun, so should be used without an article if you mean "some markup". 

```suggestion
To populate the Popup with content, add markup inside the component. You can also enable a toolbar at the bottom of the content area. To do this, declare [toolbarItems](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/) in the markup as shown in this demo. Set each item's [toolbar](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/#toolbar) property to `bottom`. To learn more about toolbar configuration, refer to the following tutorial: [Getting Started with Toolbar](/Documentation/Guide/UI_Components/Toolbar/Getting_Started_with_Toolbar/)
```